doc/lms.mysql

```diff
@@ -3670,19 +3670,19 @@ URL: %url
 ('payments', 'date_format', '%Y/%m/%d', '', 0),
 ('payments', 'default_unit_name', 'pcs.', '', 0),
 ('voip', 'default_cost_limit', '200','', 2),
-('invoices','template_file','invoice.html','',0),
-('invoices','content_type','text/html','',0),
-('invoices','cnote_template_file','invoice.html','',0),
+('invoices','template_file','FT-0100','',0),
+('invoices','content_type','pdf','',0),
```

Comment:
Gwoli ścisłości: jeśli  już zmieniasz content_type z typu MIME `text/html` to wypadałoby chyba zmienić na tym MIME właściwy dla plików `pdf`?

Reply:
https://github.com/chilek/lms/pull/2099/commits/95ff9aaad2eba83bedfd5368aa1963ec17763395